infra/Pos/Diffusion/Types.hs

```diff
@@ -77,15 +84,17 @@ dummyDiffusionLayer = DiffusionLayer
   where
     dummyDiffusion :: Applicative m => Diffusion m
     dummyDiffusion = Diffusion
-        { getBlocks          = \_ _ _ -> pure (Left (GetBlocksError "not implemented"))
+        { getBlocks          = \_ _ _ -> pure (OldestFirst [])
```

Comment:
Ok, that's the last thing. Did you eventually decide to put in "forms a proper chain" check inside?

Reply:
Yes, I'll add a comment saying this.